code/modules/clothing/accessories/holster.dm

```diff
@@ -10,22 +10,33 @@
 	rarity_value = 10
 	spawn_tags = SPAWN_TAG_HOLSTER
 	var/obj/item/holstered
+	var/sound_in = 'sound/effects/holster/holsterin.ogg' //occy edit
+	var/sound_out = 'sound/effects/holster/holsterout.ogg' //occy edit
 
 /obj/item/clothing/accessory/holster/proc/holster(obj/item/I, mob/living/user)
+
 	if(holstered && istype(user))
 		to_chat(user, SPAN_WARNING("There is already \a [holstered] holstered here!"))
 		return
 
-	if (!(I.slot_flags & SLOT_HOLSTER))
-		to_chat(user, SPAN_WARNING("[I] won't fit in [src]!"))
-		return
+	if(istype(src, /obj/item/clothing/accessory/holster/sword)) //occy edit start
```

Comment:
Nitpick, but can you change this to Occulus Edit so we have a standardized way of searching for Occulus Edited code?